### PR TITLE
Fix for accessibility issue with webform calendar control

### PR DESCRIPTION
**Pull request by elcazador**
patch prevents calendar from receiving focus anytime the enter key is pressed.

### CODE REVIEW

File: webform_calendar_aoda.patch [hunk 1]
```diff
@@ -0,0 +1,14 @@
+diff --git a/js/webform.js b/js/webform.js
+index 7daacef6..3a9101c4 100644
+--- a/js/webform.js
++++ b/js/webform.js
+@@ -84,7 +84,8 @@
+ 
+       // Prevent the calendar button from submitting the form.
+       $calendar.click(function (event) {
+-        $(this).focus();
++        // Fix for AODA when enter key pressed
```
Comment (minorOffense) [inline webform_calendar_aoda.patch]: Is there a minified version of this file we need to change too?

Comment (elcazador) [inline webform_calendar_aoda.patch]: None that I'm aware  of.  Local testing was successful with just this patch.

**Comment by minorOffense** [comment]
Only thing I'd add is a link to the issue on Drupal.org in the js comments 
``
// @see link
``

File: webform_calendar_aoda.patch (hunk 1 above)
Comment (elcazador) [inline webform_calendar_aoda.patch]: Good point.  Added it from the issue comment.

**Pull request merged**